config/cypress.config.js

```diff
@@ -197,4 +197,13 @@ const cypressConfig = {
   },
 };
 
-module.exports = defineConfig(cypressConfig);
+module.exports = {
+  ...defineConfig(cypressConfig),
+
+  component: {
+    devServer: {
+      framework: 'react',
+      bundler: 'webpack',
+    },
+  },
+};
```

Comment:
Does this mean that we can use cypress to do their 'component testing' now as well?